nautobot/core/tables.py

```diff
@@ -38,6 +40,14 @@ def __init__(self, *args, user=None, hide_hierarchy_ui=False, **kwargs):
         # Add custom field columns
         model = self._meta.model
 
+        if model.is_static_group_associable_model:
+            self.base_columns["static_group_count"] = LinkedCountColumn(
+                viewname="extras:staticgroup_list",
+                url_params={"member_id": "pk"},
```

Comment:
I think you brought this up in dsu. How difficult would it be to add content type to this? I know it's astronomically unlikely for the uuids to collide, so definitely not blocking

Reply:
A little tricky here because `LinkedCountColumn` expects each value in the `url_params` (e.g. `"pk"`) to be an instance attribute that it can `getattr()` on. So just adding `"content_type": model._meta.label_lower` or even `"content_type": "_meta.label_lower"` wouldn't work as one might expect. Might be able to make LinkedCountColumn smart enough to handle nested attributes though...

Reply:
> Might be able to make LinkedCountColumn smart enough to handle nested attributes though...

Probably not worth it for this use case